connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectCluster.java

```diff
@@ -126,11 +154,46 @@ public void startConnect() {
         putIfAbsent(workerProps, KEY_CONVERTER_CLASS_CONFIG, "org.apache.kafka.connect.storage.StringConverter");
         putIfAbsent(workerProps, VALUE_CONVERTER_CLASS_CONFIG, "org.apache.kafka.connect.storage.StringConverter");
 
-        for (int i = 0; i < connectCluster.length; i++) {
-            connectCluster[i] = new ConnectDistributed().startConnect(workerProps);
+        for (int i = 0; i < numInitialWorkers; i++) {
+            connectCluster.add(new ConnectDistributed().startConnect(workerProps));
         }
     }
 
+    /**
+     * Get the workers that are up and running.
+     *
+     * @return the list of online workers
+     */
+    public Collection<String> activeWorkers() {
+        ObjectMapper mapper = new ObjectMapper();
+        return connectCluster.stream()
+                .map(Connect::restUrl)
+                .map(URI::toString)
+                .map(url -> {
+                    try {
+                        mapper.readerFor(ServerInfo.class).readValue(executeGet(url));
+                        return url;
+                    } catch (IOException e) {
+                        // Worker failed to respond. Consider it's offline
+                        return null;
+                    }
+                })
+                .filter(Objects::nonNull)
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Get the provisioned workers.
```

Comment:
Nit: this returns the _URLs_ of the provisioned workers in this cluster. Would be good to state this explicitly in the JavaDoc, including in the `@return` statement

Reply:
done